index.html

```diff
@@ -0,0 +1,362 @@
+    </div>
+
+    <div class=" d-block d-sm-block d-md-none" style="position: relative;">
+      <div style="width: 100%; height: 450px;">
+      </div>
+
+      
+      <div class="azul container-fluid p-5 text-center" style="background-color:#0A83ED">
+      
+        <img id="bag"  src="../bluepads/assets/Desktop/getbetterimage.png" alt="getbetterimage">
+        <h3 style="color: white;"> <br><br><br><br> No importa si es un concierto en vivo o una sesión de grabación en estudio, BluePads te ayudarán a crear el sonido perfecto.</h3>
+      </div>
+</div>
+<div class=" d-block d-sm-block d-md-none" style="background-color: #0A83ED; height: 0%;">
+<div class="col recgrisc d-block d-sm-block d-md-none"> <br>
+  <div class=" mt-3 ml-5" >
+  <h2 style="font-weight: 900;">BLUEPADS</h2>
+  <H1 style="font: weight 900px; font-size: 84px;">$59</H1>
+<p style="color: black;line-height: 22px;"> <b> Incluye 6 piezas</b> <br>
+<font style=" color: #939393; font-size: 16px; ">(35mm x 35mm x 1mm de grosor)</font></p>
+<p style="font-size: 14px; line-height: 22px;"> <font style=" color: black"> <b>Construcción </b></font> <br> Adhesivo: Acrílico <br>Colores: Azul traslucido <br>Revestimiento: PET film (White) 
+<br><br>
+<font style=" color: black"> <b>Grosor 1.0mm</b>  </font>
+</p>
+</div>
+<button type="button" class="ver-caracteristicasc btn btn-primary button-nav mt-5 mb-5 mx-auto"><p style="font-size: 25px; color: white; font-weight: 900; margin: auto;" >COMPRAR</p></button>   
+
+</div>
+</div>
+
+
+
+      <div class="container-fluid d-block d-sm-block d-md-none" style="background-color:#0A83ED">
+        <div class="row pt-5 pb-5">
+          <div class="col">
+          <a href="#">
+            <img src="../bluepads/assets/responsiva/whitelogox2.png" alt="white-logo">
+          </a>
+         </div>
+         <div class="col pl-5 ml-5">
+          <a class="ml-5" href="#">
+                <img class="footerface" src="../bluepads/assets/responsiva/vector.png" alt="facebook">
+              </a>
+            <a class="ml-5" href="#">
+              <img src="../bluepads/assets/responsiva/ic-instagram.png" alt="instagram">
+            </a>
+          </div>
+          </div>
+        </div>  
+
+        
+    
+         
+     
+    
+    
+
+
+
+
+    <nav class="navbar navbar-light border-bottom d-none d-sm-none d-md-block">
+      <div class="container">
+        <a class="navbar-brand" href="#">
+          <img class="d-flex align-text-top" src="../bluepads/assets/Desktop/logo-header.png" alt="bluepads-logo" >
+        </a>
+        <div class="d-flex justify-content-end">
+        <button type="button" class=" rounded-pill btn btn-primary ml-5 text-nowrap" style="font-size: 16px; font-weight: 900;">COMPRAR</button>
+        <a class="ml-5" href="#">
```

Comment:
todo el file tiene pedos de intentación, esto ayuda un chingo a mantener el código organizado y que sea fácil de ver lo que está pasado, acá hay un articulo chido donde explican que es indentar https://www.granneman.com/webdev/coding/formatting-and-indenting-your-html, pero lo más verga es que es 2021 y el VS Code lo puede hacer por ti, acá las instrucciones https://linuxpip.org/auto-indent-vscode/